src/routes/projects/[id([0-9]+)]/index.svelte

```diff
@@ -236,15 +236,20 @@
 
   const reportPerformanceModal = {
     open: false,
+    activity: null,
+    application: null,
+    report: null,
     show({ detail }) {
       reportPerformanceModal.activity = detail.activity;
       reportPerformanceModal.application = detail.application;
+      reportPerformanceModal.report = detail.report;
       reportPerformanceModal.open = true;
     },
     async submitReport({ detail }) {
       try {
-        const { value, activity, application } = detail;
-        application.reports.push(await api.json(api.post(
+        const { value, activity, application, report } = detail;
+        const apiCall = report == null ? api.post : api.patch;
```

Comment:
Could just add `PUT` method in backend, but it doesn't matter much)

Reply:
Let's be consistent